core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/action/ActionConfiguration.java

```diff
@@ -161,7 +162,8 @@ public boolean exists(ActionKey actionKey) {
                 + getMetadataRepository().getTableNameByLabel("Actions")
                 + " WHERE ACTION_ID = " + SQLTools.getStringForSQL(actionKey.getActionId()) +
                 " AND SCRIPT_ID = " + SQLTools.getStringForSQL(actionKey.getScriptKey().getScriptId()) +
-                " AND SCRIPT_VRS_NB = " + SQLTools.getStringForSQL(actionKey.getScriptKey().getScriptVersion()) + ";";
+                " AND SCRIPT_VRS_NB = " + SQLTools.getStringForSQL(actionKey.getScriptKey().getScriptVersion()) +
+                " AND DELETED_AT = 'NA';";
         CachedRowSet cachedRowSet = getMetadataRepository().executeQuery(query, "reader");
         return cachedRowSet.size() >= 1;
     }
```

Comment:
deleteStatement below should use ALL fields part of the ActionKey. So also deletedAt